features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/TableView/TableView.jsx

```diff
@@ -443,7 +444,11 @@ class TableView extends React.Component {
         }
         if (apisAndApiProducts.length === 0 && !query) {
             return (
-                <>
+                <ApiListContextProvider
```

Comment:
Why do we need to introduce a context here ?
Can't we do this check inside the `RestAPIMenu.jsx ` ?

Reply:
fixed with 1bc0c49